code/datums/diseases/advance/symptoms/symptoms.dm

```diff
@@ -21,12 +21,20 @@ var/global/const/SYMPTOM_ACTIVATION_PROB = 5
 
 /datum/symptom/New()
 	var/list/S = list_symptoms
+
+	//Randomize Symptom Stats
+	stealth += rand(-1,1)

```

Comment:
Not sure how that is possible, the advance diseases' New() proc has a disease parameter which it copies the symptoms from by instantiating new symptom datums, which would change the levels in the New().